### ISSUE TITLE
Increase the salience of the 'sign in' button for guest users

### ISSUE THREAD

**Issue by BethanOFN** [openfoodfoundation/wishlist#405]
## What is the need / problem?
We want to maximise the number of users creating an account with us. Feedback from a behavioural consulting firm was that the 'sign in' button was not salient enough on the homepage.

## Which type of users does this problem affect (and how many, if known)?
All guest users

## Potential solutions that will solve the problem?
Bolden, increase the size and change the colour of the text of the 'sign in' button to OFN orange:
<img width="155" alt="Screenshot 2023-02-13 at 14 28 09" src="https://user-images.githubusercontent.com/100226210/218485012-1fba5f4c-67f9-4be9-81f3-a7a2cbcb41a4.png">
We want the 'sign in' button to be as salient, if not more salient, than the basket button for guest users.

## Additional context
Proposal via BVA Nudge

**Comment by lin-d-hop** [comment]
Extending on my last comment on #403 
We could explore all these nudge improvements together as funded features if you would like?

**Comment by BethanOFN** [comment]
> Extending on my last comment on #403 We could explore all these nudge improvements together as funded features if you would like? But 🤞 finger crossed they are all papercuts :)

Yep all papercuts, once I get them all in then can look at funded feature potential 👍
